app/models/results.js

```diff
@@ -92,6 +92,19 @@ const ResultSchema = new Schema({
 });
 
 ResultSchema.set('toObject', {virtuals: true});
+ResultSchema.set('toJSON', {
```

Comment:
Will try to cope without it

**note:** similar approach is used in models/testcase.js,